tests.log

```diff
@@ -0,0 +1,111 @@
+**************** vetting go code **********************
```

Comment:
Please clean up added files. `tests.log` & `~/Documents`

Reply:
I have no idea how files outside the repo got added!  Thanks for flagging this, I'll clean it up.